fields/types/localfiles/LocalFilesField.js

```diff
@@ -52,13 +51,22 @@ var LocalFilesFieldItem = React.createClass({
 
 		return (
 			<FormField>
-				<img key="file-type-icon" className="file-icon" src={Keystone.adminPath + '/images/icons/32/' + iconName + '.png'} />
-				<FormInput key="file-name" noedit className="field-type-localfiles__filename">
-					{filename}
-					{this.props.size ? ' (' + bytes(this.props.size) + ')' : null}
-				</FormInput>
-				{note}
-				{this.renderActionButton()}
+				<div style={{display: 'flex'}}>
+					{isPicture &&
+						<div style={{display: 'flex', width: 150, marginRight: 10}}>
+							<img style={{width: '100%', height: '100%'}} key="file-type-icon" className="file-icon" src={`/files/${listKey}/${itemId}/${filename}/${originalname}`}/>
+						</div>
+					}
+					<div style={{display: 'flex', flexDirection: 'column', justifyContent: 'space-between', alignItems: 'flex-start', minHeight: 100, width: '100%'}}>
+						<FormInput key="file-name" noedit className="field-type-localfiles__filename">
+							<a href={`/files/${listKey}/${itemId}/${filename}/${originalname}`}>{originalname}</a>
```

Comment:
and of course this either needs to be a generic file download endpoint in Keystone (with access control) or a configurable url (via a function with the above variables as attributes)